lib/ansible/plugins/action/net_put.py

```diff
@@ -25,6 +25,7 @@
 import sys
 import re
 
+from ansible.errors import AnsibleError
 from ansible.module_utils._text import to_text, to_bytes
 from ansible.module_utils.connection import Connection
 from ansible.plugins.action.network import ActionModule as NetworkActionModule
```

Comment:
`net_put` module is still not inheriting directly from class `ActionBase` and continuing to inherit from `action.network`.